device_doctor/lib/src/ios_debug_symbol_doctor.dart

```diff
@@ -170,6 +175,24 @@ class RecoverCommand extends Command<bool> {
     }
     return true;
   }
+
+  /// Delete all symbols by deleting the `iOS DeviceSupport` directory.
+  /// Xcode will regenerate this folder and symbols for connected devices
+  /// when Xcode is opened.
+  void _deleteSymbols() {
+    final String? home = platform.environment['HOME'];
+    if (home == null) {
+      logger.warning('\$HOME path was not found');
+      return;
+    }
+    final Directory deviceSupportDirectory = fs.directory('$home/Library/Developer/Xcode/iOS DeviceSupport');
+    if (!deviceSupportDirectory.existsSync()) {
+      logger.warning('iOS Device Support directory was not found at ${deviceSupportDirectory.path}');
+      return;
+    }
+    logger.info('Deleting iOS DeviceSupport...');
+    deviceSupportDirectory.deleteSync(recursive: true);
```

Comment:
What should we do if this delete fails (maybe Xcode or some daemon is running and has a file under it open)? My first thought would be log it and keep going, as I'm not sure if it's really actionable.

Reply:
Yeah I'm not sure it's really actionable either. I added a try catch to log it out